src/index.html

```diff
@@ -1,15 +1,157 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <h1>HTML Form</h1>
+    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="POST"
+          autocomplete="off">
+      <fieldset class="block_1">
+        <legend> Personal information </legend>
+        <label for="surname" class="block_1_item">
+
+          Surname:
+          <input
+        type="text"
+        name="surname"
+        id="surname">
+        </label>
+        <label for="name" class="block_1_item">
+          Name
+          <input
+        type="name"
+        name="name"
+        id="name">
+        </label>
+        <label for="age" class="block_1_item">
+      How old are you
+          <input type="number"
+        name="age"
+        id="age"
+        value="1"
+        min="1"
+        max="100">
```

Comment:
```suggestion
          name="age"
          id="age"
          value="1"
          min="1"
          max="100"
        >
```

Fix formatting everywhere in your code